### DISCUSSION TITLE
One stop place for fflib-apex-common documentation

### DISCUSSION THREAD

**Discussion by wimvelzeboer** [apex-enterprise-patterns/fflib-apex-common#345] [Ideas]
Hi,
There is currently not a really good online one-stop place for people wanting to learn more about AEP.
We currently have the book from @afawcett and the code examples [fflib-apex-common-samplecode](https://github.com/apex-enterprise-patterns/fflib-apex-common-samplecode).

The book is great but its a long read and the fflib-apex-common-samplecode is directly a deep dive into code. Many developers are looking for something in between. Something like a one-pager with a quick explanation of what it is, what it is for, how to use it and some basic examples. It can then have links to chapters in @afawcett book and to the samplecode.

I have been working on something like this for the company I work for, but I would like to add it to the AEP repository, but where would be the best place?
Add a folder named docs in flib-apex-common?
Or extend the fflib-apex-common-samplecode with documentation?
Maybe a complete separate fflib-docs repository?

What are your thoughts?
@daveespo @stohn777 @ImJohnMDaniel @afawcett

**Comment by daveespo** [comment]
Hi @wimvelzeboer  -- great question and we did think about this a while ago but it didn't get far. 

This repository is a GH Pages repo https://github.com/apex-enterprise-patterns/apex-enterprise-patterns.github.io

We'd gladly review and accept PRs against this repo for new/revised content. If you have some content that you have already authored and can share with us directly, that would help ensure that we're not duplicating effort. 

There are some other sources of content out there that we should also consider linking to or directly incorporating it and we'll be doing so over this summer.

Also, we'll gladly accept suggestions for pages that suggest best practices/patterns for implementing common solutions.
